dojo/tools/veracode_sca/parser.py

```diff
@@ -65,17 +65,10 @@ def _get_findings_json(self, file, test):
                 cvss_score = vulnerability.get("cvss3_score")
             severity = self.__cvss_to_severity(cvss_score)
 
-            description = 'This library has known vulnerabilities.\n'
-            description += \
-                "**CVE:** {0} ({1})\n" \
-                "CVS Score: {2} ({3})\n" \
-                "Project name: {4}\n" \
-                "Title: \n>{5}" \
+            description = \
+                "Project name: {0}\n" \
```

Comment:
I'm talking about this text

Reply:
Yes but I have not changed anything or added anything, just removed some other text. What should it look like?

Reply:
I mean, you can wrap part of this text in localization

Reply:
Have other parsers made this change yet?